docs/guides/setup_cp_and_deploy_sidecar.md

```diff
@@ -0,0 +1,157 @@
+---
+page_title: "Create a sidecar and repository in Cyral control plane and bring up the sidecar on AWS"
+---
+
+Use the following code to get a basic scenario with sidecar and repository set in the Cyral 
+control plane and then bring this sidecar up on your AWS account.
+
+By running this example you will have a fully functional sidecar on your AWS account to
+protect your database. Read the comments and update the necessary parameters as instructed.
+
+See also the [Cyral sidecar module for AWS](https://registry.terraform.io/modules/cyralinc/sidecar-aws/cyral/latest)
+for more details on how the sidecar is deployed to AWS.
+
+```hcl
+terraform {
+  required_providers {
+    cyral = {
+      source = "cyralinc/cyral"
+      version = ">= 2.4.0"
+    }
+  }
+}
+
+locals {
+    # Replace [TENANT] by your tenant name. Ex: mycompany.app.cyral.com
+    control_plane = "[TENANT].app.cyral.com"
+
+    sidecar = {
+        # The sidecar name prefix is a unique name that is used by the sidecar module to create
+        # resources in the target AWS account. For this reason, we use 'cyral-zzzzzz' where
+        # zzzzzz are the last 6 digits of the sidecar id created in the control plane. This
+        # explanation is purely informational and we advise you to keep this variable as is.
+        sidecar_name_prefix = "cyral-${substr(lower(cyral_sidecar.mysql_sidecar.id), -6, -1)}"
+
+        # If you would like to use other log integration, download a new template from the UI
+        # and copy the log integration configuration or follow this module documentation.
+        log_integration = "cloudwatch"
+        # Set the desired EC2 instance type for the auto scaling group.
+        instance_type = "t3.medium"
+        # Set to true if you want a sidecar deployed with an internet-facing load
+        # balancer (requires a public subnet).
+        public_sidecar = false
+
+        # Set the AWS region that the sidecar will be deployed to
+        region = ""
+        # Set the ID of VPC that the sidecar will be deployed to
+        vpc_id = ""
+        # Set the IDs of the subnets that the sidecar will be deployed to
+        subnets = [""]
+
+        # Set the allowed CIDR block for SSH access to the sidecar
+        ssh_inbound_cidr = ["0.0.0.0/0"]
+        # Set the allowed CIDR block for database access to the sidecar
```

Comment:
Suggest changing "to" to "through" like this: "Set the allowed CIDR block for database access through the sidecar"